README.md

```diff
@@ -27,6 +28,34 @@ class MyQuerysetView(PostgresViewFromQueryset):
         )
 ```
 
+* The package allows reading from a view using django ORM. For a view to be readable, it has to be inherited from 
+ one of the readable view abstractions - `ReadableViewFromQueryset` or `ReadableViewFromSQL`. It would look like this:
+```python
+
+class MyReadableView(ReadableViewFromQueryset):
+    
+    def get_queryset(self):
```

Comment:
Needs to be a `@classmethod` now, right?  The other examples prob need updating as such too